### PR TITLE
geth (ropsten -> rinkeby & 1.9.12 -> 1.9.13), go (1.14 -> 1.14.2)

### PR DESCRIPTION
**Pull request by kilrau**
I am running this branch since a while and rinkeby is working fine. For me it finds peers quickly and went from 0 to 100% in about 36h on my Mini PC (0 to 99.99% in some hours). I think we can merge this now already to make it easier for some more of us to sync rinkeby, since raiden is refusing to connect to a local geth anyways (only infura works).

### CODE REVIEW

File: images/geth/1.9.12/rinkeby-peers.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
+enode://8f44b492b956f300ae786f3d55280dae3189b670ab7ba4c6020279930c5852cdebf688ee92e1f7961d9234857d3b7ba98a993e54bb4cf1fb900da5ec4acd026f@54.202.114.179:30303
```
Comment (michael1011) [inline images/geth/1.9.12/rinkeby-peers.txt:1]: Do we *really* need these hardcoded bootnodes? I have had no issues getting my nodes synced without any

**Comment by kilrau** [comment]
Disabled bootnodes: please give this a run @peartobear @raladev

File: images/geth/1.9.13/entrypoint.sh [hunk 2]
```diff
@@ -22,7 +22,8 @@ if [[ $CUSTOM_ANCIENT_CHAINDATA == "true" ]]; then
 fi
 
 if [[ $NETWORK == "testnet" ]]; then
-  OPTS+=("--testnet")
+# 3=ropsten 4=rinkeby for more see: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md#list-of-chain-ids
+  OPTS+=("--networkid=4")
```
Comment (michael1011) [inline images/geth/1.9.13/entrypoint.sh:26]: Use `--rinkeby` instead of `--networkid=4`

Comment (kilrau) [reply images/geth/1.9.13/entrypoint.sh:26]: Can do, but reason?

**Comment by kilrau** [comment]
Upgraded to [geth 1.9.13](https://github.com/ethereum/go-ethereum/releases/tag/v1.9.13) and in that course also bumped `go` to 1.14.2 which should make things much more stable with geth according to the release notes. Also fixed a stupid mistake of mine so that this is finally syncing rinkeby and not ropsten.

Works well for me and would appreciate at least one more person running this before we merge.

File: images/geth/1.9.13/entrypoint.sh (hunk 2 above)
Comment (kilrau) [reply images/geth/1.9.13/entrypoint.sh]: Recap: reason was readability and bootnodes that might hide behind the `--rinkeby` flag, but not behind the `--networkid` flag.

Switched to  `--rinkeby`.

**Comment by kilrau** [comment]
I ran this branch on mainnet and works fine. I'll go ahead and merge this.

**Pull request merged**